src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
       private ColorSupplier colorSupplier = new ColorSupplier();
```
замість наслідування краще використати композицію (коли один клас має посилання на інший). Так ти можеш викорисати багато класів